config/ui-extensions/istios/translations

```diff
@@ -11,6 +11,7 @@ en:
   config.authorizers.headers.toUpstream.onAllow: Forward To Upstream
   config.authorizers.headers.toDownstream.onAllow: Forward To Downstream On Allow
   config.authorizers.headers.toDownstream.onDeny: Forward To Downstream On Deny
+  config.gatewayExternalTrafficPolicy: Gateway external traffic policy
```

Comment:
```suggestion
  config.gatewayExternalTrafficPolicy: External traffic policy for Istio Ingress Gateway
```

Reply:
alternatively: `External traffic policy for Gateway`

Reply:
The field has description defined under the ℹ️ icon, I would not make the UI name longer
<img width="694" alt="Screenshot 2024-04-22 at 14 22 16" src="https://github.com/kyma-project/istio/assets/103247439/e6bc138e-501b-44cd-955b-6f58eff8c68b">